client/src/components/pages/Activities/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Card from '../../common/Card';
+import './index.css';
+
+const Activities = () => {
+  return (
+    <div className="ActivityName">
+      <p className="ActivityName__tiltle">Class Activities</p>
```

Comment:
typo in title; also pls use either camelCase or kebab-case for class names (pick one as a team and stay consistent): `ActivityName` -> `activityName`

Reply:
ok , We will do that.